client/client-multi/client-c/tools/kaa_encryption/rsa_key_gen/kaa_rsa_key_gen.h

```diff
@@ -0,0 +1,110 @@
+/*
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @file kaa_rsa_key_gen.h
```

Comment:
`@file` is sufficient 
Ref: https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdfile
> If the file name is omitted (i.e. the line after \file is left blank) then the documentation block that contains the \file command will belong to the file it is located in.